README.md

```diff
@@ -17,7 +19,9 @@ In a virtualenv, run:
     python setup.py install
     FLASK_APP=hello flask run
 
-## Tasks
+## Tasks to complete
+
+Now you have the app running, it's your turn to work :) 
```

Comment:
I'd prefer not to rely on smiles. Sounds less serious and in this case, I want to give the impresion of a serious company.

Reply:
Trust me, asking people to Terraform a Fargate cluster and set a CD pipeline sounds serious. 
But I agree, it came without me really noticing.